recipes/larray-editor/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "larray-editor" %}
+{% set version = "0.26.1" %}
+{% set sha256 = "57aafcb53fdeafe32b61d2a52185926706c048df028fbf76ecb7a86f76f7fb49" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/larray-project/larray-editor/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I'm pretty sure that if you include a script here then the separate build script will not run...given that you have build steps beyond setup.py, you should probably delete this line.